src/VirtoCommerce.Xapi.Core/ModuleConstants.cs

```diff
@@ -48,13 +48,73 @@ public static class General
                     DefaultValue = true
                 };
 
+                public static SettingDescriptor PageTitleWithStoreName { get; } = new SettingDescriptor
+                {
+                    Name = "Frontend.PageTitleWithStoreName",
+                    ValueType = SettingValueType.Boolean,
+                    GroupName = "Virto Commerce Frontend",
+                    DefaultValue = true,
+                    IsPublic = true
+                };
+
+                public static SettingDescriptor PageTitleStoreNameAlign { get; } = new SettingDescriptor
+                {
+                    Name = "Frontend.PageTitleStoreNameAlign",
+                    ValueType = SettingValueType.ShortText,
```

Comment:
could it be dropdown with two options?
start, end

Reply:
Resolved
![image](https://github.com/user-attachments/assets/64dc752e-7b30-43cc-93f5-4baa0bddec4c)